src/beans/canvasjs-2.3.2/license.txt

```diff
@@ -0,0 +1,9 @@
+*
+* @preserve CanvasJS HTML5 & JavaScript Charts - v2.3.2 GA - https://canvasjs.com/ 
+* Copyright 2018 fenopix
+*
+*  --------------------- License Information --------------------
+* CanvasJS is a commercial product which requires purchase of license. Without a commercial license you can use it for evaluation purposes for upto 30 days. Please refer to the following link for further details.
+*     https://canvasjs.com/license/

```

Comment:
I don't think I have money to afford this library unless we are going to remove this feature after 30 days.

Reply:
[vx](https://github.com/hshoff/vx) would be a good alternative.